examples/classic/kubernetes-private.classic.json

```diff
@@ -0,0 +1,38 @@
+{
+  "apiVersion": "vlabs",
+  "properties": {
+    "orchestratorProfile": {
+      "orchestratorType": "Kubernetes",
+      "kubernetesConfig": {

```

Comment:
@lachie83 , IsCustomVNET func in types.go and its usage in this file [{{if .IsCustomVNET}}] is an example of how this can for nopublic ip case as well.